boards/arduino-common/include/board.h

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (C) 2014 Freie Universität Berlin, Hinnerk van Bruinehsen
+ *               2016 Laurent Navet <laurent.navet@gmail.com>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    boards_arduino-common
```

Comment:
The doxygen warning triggering a Murdock error comes from this line (try `make doc` at the root of riot repository and look at the first warning). 
To fix it, you can change it like this:
```C
* @defgroup    boards_arduino-common Arduino common
```